gruntfile.js

```diff
@@ -461,7 +456,7 @@ module.exports = function (grunt) {
         command: [
           'docker stop portainer',
           'docker rm portainer',
-          'docker run --privileged -d -p 9000:9000 -v /tmp/portainer:/data -v /var/run/docker.sock:/var/run/docker.sock --name portainer portainer --no-analytics'
+          'docker run --privileged -d -p 9000:9000 -v $(pwd)/dist/public:/public -v /tmp/portainer:/data -v /var/run/docker.sock:/var/run/docker.sock --name portainer portainer --no-analytics'
```

Comment:
Instead of altering the default value as I commented above, why not use the `-a /public` flag here?